elitzur-core/src/test/scala/com/spotify/elitzur/validators/DynamicRecordValidatorTest.scala

```diff
@@ -90,29 +90,23 @@ class DynamicRecordValidatorTest extends AnyFlatSpec with Matchers {
     ) shouldEqual 1
   }
 
-//  it should "validate a repeated value" in {
-//    val label = "label"
-//    val className = "com.spotify.DynamicClass"
-//    implicit val metricsReporter: MetricsReporter = DynamicRecordValidatorTest.metricsReporter()
-//    val repeatedValidator = implicitly[Validator[Seq[NonNegativeLong]]]
-//    val recordValidator = DynamicRecordValidator(
-//      Array(repeatedValidator).asInstanceOf[Array[Validator[Any]]], Array(label))
-//    recordValidator.validateRecord(
-//      Unvalidated(Seq(Seq(NonNegativeLong(1L), NonNegativeLong(-1L))).asInstanceOf[Seq[Any]]),
-//      outermostClassName = Some(className)
-//    )
-//    print(metricsReporter.toString)
-//    metricsReporter.asInstanceOf[TestMetricsReporter].getValid(
-//      className,
-//      label,
-//      NonNegativeLongCompanion.validationType
-//    ) shouldEqual 1
```

Comment:
would the test be better if we assert that this should be 0 ?

Reply:
hmm there is a valid one though, right so it should be 1 because of the `1L`?

Reply:
hmm yes but it's in the array with the invalid case. my understanding is that:

[1L, -1L] goes into the dynamic record validator as an array element, it hits the valdiatorLoop which basically maps it to [true, false] which then evaluates to false and that's why we only see one 1 count towards the invalid case and nothing towards valid case